account_invoice_pricelist/views/account_invoice_view.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<odoo>
+    <record id="view_account_move_customer_form" model="ir.ui.view">
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account.view_move_form" />
+        <field name="arch" type="xml">
+            <button name="action_invoice_sent" position="before">
```

Comment:
Should add the group "product.group_product_pricelist" for this button.
Same for the field, filter below

Reply:
Good idea